flink-cdc-cli/src/test/java/org/apache/flink/cdc/cli/CliFrontendTest.java

```diff
@@ -145,6 +147,86 @@ void testPipelineExecuting() throws Exception {
         assertThat(executionInfo.getDescription()).isEqualTo("fake-description");
     }
 
+    @Test
+    void testPipelineExecutingWithFlinkConfig() throws Exception {
+        // the command line arguments to submit job to exists remote host on yarn session
+        CliExecutor executor =
+                createExecutor(
+                        pipelineDef(),
+                        "--flink-home",
+                        flinkHome(),
+                        "--global-config",
+                        globalPipelineConfig(),
+                        "-D",
+                        "execution.target= yarn-session",
+                        "-D",
+                        "rest.bind-port =42689",
+                        "-D",
+                        "yarn.application.id=application_1714009558476_3563",
+                        "-D",
+                        "rest.bind-address=10.1.140.140");
+        Map<String, String> configMap = executor.getFlinkConfig().toMap();
+        assertThat(configMap)
+                .containsEntry("execution.target", "yarn-session")
+                .containsEntry("rest.bind-port", "42689")
+                .containsEntry("yarn.application.id", "application_1714009558476_3563")
+                .containsEntry("rest.bind-address", "10.1.140.140");
+    }
+
+    @Test
+    void testPipelineExecutingWithUnValidFlinkConfig() throws Exception {
+        IllegalArgumentException exception =
+                Assert.assertThrows(
```

Comment:
It's not recommended to mix AssertJ and JUnit style Assert together. Prefer using `Assertions.assertThatThrownBy`, `isExactlyInstanceof`, and `hasMessage`